src/realm/sync/config.cpp

```diff
@@ -137,6 +146,8 @@ SimplifiedProtocolError get_simplified_error(sync::ProtocolError err)
         case ProtocolError::invalid_schema_change:
         case ProtocolError::server_file_deleted:
         case ProtocolError::user_blacklisted:
+        case ProtocolError::object_already_exists:
+        case ProtocolError::server_permissions_changed:
```

Comment:
I'm not sure that these should cause a client reset. At least, I expect it will be incorrect to try to use recovery mode for the `object_already_exists` error. But we can deal with this at a later point since client reset with FLX isn't even working yet.

Reply:
@mpobrien said these should cause a client reset.

Reply:
ok, let's visit this later when implementing recovery mode